app/src/main/java/org/merakilearn/ui/profile/ProfileViewModel.kt

```diff
@@ -51,24 +52,22 @@ class ProfileViewModel(
                 copy(showEditProfileLayout = true, showUpdateProfile = true)
             }
             is ProfileViewActions.ShareFile -> shareFile(action.file)
-            is ProfileViewActions.UpdateServerUrlClicked -> _viewEvents.setValue(ProfileViewEvents.ShowUpdateServerDialog(settingsRepo.serverBaseUrl))
```

Comment:
Why have we removed this code?

Reply:
In the new design, it was not present so I thought it should be removed.